docs/source/data-access-api/openapi.yml

```diff
@@ -153,6 +155,24 @@ paths:
           required: false
           schema:
             type: string
+        - in: query
+          name: filename
+          description: WIP -- The full filename of the desired file (e.g. ``imap_mag_l1a_burst_20240105_20240105_v01-01.pkts``).
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: N_days_ago
+          description: WIP -- Returns all files acquired in the last 'N' days (with 'N' representing a numeric value).
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: today
+          description: WIP -- Returns all files acquired in the last 24 hours.
+          required: false
+          schema:
+            type: string
```

Comment:
I would prefer to add these in as we implement these features so we get in the habit of updating the docs in the same PRs that implement said features. Also, that would avoid a situation in which the docs say 'we are going to add this feature' but we don't actually get around to implementing it for several months, which could be a bad look.

Reply:
Totally agree with you!